lisa/features/infiniband.py

```diff
@@ -146,6 +149,8 @@ def _get_ib_device_names(self) -> List[str]:
     def _get_mofed_version(self) -> str:
         node = self._node
         default = "5.4-3.0.3.0"
+        if self._is_legacy_device():
+            return "4.9-5.1.0.0"
```

Comment:
Is it possible to parse out the version directly?

Reply:
This is not parsing the version on the VM, this is making a decision for which OFED drivers to install.

Most stable is 5.4 (LTS), old cards require 4.9 LTS, newest distros may require newer versions